src/cirrus/plugins/publishers/documentation.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+"""
+_documentation_
```

Comment:
originally it did match, then i changed the plugin name but hadn't thought to change the module name, i'll do that